index.rst

```diff
@@ -58,6 +58,13 @@ To install the LSST Simulation software, such as MAF, please follow the `LSST Si
 
    install/index
 
+Frameworks
+==========
+
+.. toctree::
```

Comment:
To avoid putting section headers from the FAQ on the homepage, I think we should use the max depth flag here:

```suggestion
.. toctree::
   :maxdepth: 2
```